src/s2wasm.h

```diff
@@ -935,8 +981,13 @@ class S2WasmBuilder {
       } else if (match("end_block")) {
         bstack.back()->cast<Block>()->finalize();
         bstack.pop_back();
-      } else if (match(".LBB")) {
-        s = strchr(s, '\n');
+      } else if (peek(".LBB")) {
+        // FIXME legacy tests: it can be leftover from "loop" or "block", but it can be a label too
```

Comment:
.Lfunc_end is still in the .s file, but is emitted after the .end_func so s2wasm doesn't consider it to be part of the function body. .LBB is still present and isn't referenced from the actual code, though it probably is referenced by debug info, which is probably why this patch needs to parse it.